.github/workflows/internal-archive-release.yml

```diff
@@ -30,9 +30,15 @@ on:
         type: string
         required: true
       ref:
-        description: "The ref to use (leave empty to use main)"
+        description: "The ref to use (default to main)"
         type: string
+        default: "main"
```

Comment:
If this is required, then the default wouldn't get used. I think this should be `required: false` below. The input is not required to be provided, even if it's required below.

Reply:
Yeah it's a called-only workflow so I'll take you on that here!